actions/gallery_actions.js

```diff
@@ -152,63 +93,17 @@ export const getPhotosFromCameraroll = () => async (dispatch, getState) => {
                     width: item.node.image.width,
                     lat: item.node.location.latitude,
                     lon: item.node.location.longitude,
-                    timestamp: item.node.timestamp,
+                    date: item.node.timestamp,
```

Comment:
Is this date or timestamp? Comment would be good here

Reply:
I changed the `key` from timestamp to date SO that both gallery and camera image have same key

Reply:
If the value is timestamp, then the key should also be timestamp?

Reply:
I made it date because API was `date` and generally timestamps are used for dates.